src/rviz/viewport_mouse_event.h

```diff
@@ -54,8 +54,13 @@ class ViewportMouseEvent
     : panel(p)
     , viewport(vp)
     , type(e->type())
+#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)
```

Comment:
I guess the new API is only available since Qt 6?
```suggestion
#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
```

Reply:
> In addition to the existing imported targets, Qt 5.15 introduced versionless targets. That is, to link against [Qt Core](https://doc.qt.io/qt-6/qtcore-index.html) one can both reference Qt6::Core, or Qt::Core

https://doc.qt.io/qt-6/cmake-qt5-and-qt6-compatibility.html

Reply:
Ok. But this is unrelated to the API change required here, i.e. `QMouseEvent::[x|y]()` vs. `QMouseEvent::position().[x|y]()`.

Reply:
Sorry, I wasn't reading the context very carefully and have a cold ...

> This function was introduced in Qt 5.14.

https://doc.qt.io/qt-5/qwheelevent.html#position

I assume the previous way was deprecated in 5.15 but haven't looked it up. Actually I just copy pasted from line 76/81 below

Reply:
In contrast to those lines we are dealing with a `QMouseEvent` here, not a `QWheelEvent`. Without the proposed change, I had a compiler failure with Qt 5.15.